bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py

```diff
@@ -96,15 +99,20 @@ def make_bank_payment(source_name, target_doc=None):
 	#Assigning party type as supplier
 	def set_supplier(source_doc,target_doc,source_parent):
 		target_doc.party_type="Supplier"
-	
+		target_doc.reconcile_action="Manual Reconcile"
+		target_doc.append('payment_references',{
+			'reference_name': source_doc.name,
+			'reference_doctype': 'Purchase Invoice',
+			"total_amount": source_doc.rounded_total,
+			"outstanding_amount":source_doc.outstanding_amount
+		})
 	from frappe.model.mapper import get_mapped_doc
 	doclist = get_mapped_doc("Purchase Invoice", source_name,{
 		"Purchase Invoice": {
 			"postprocess": set_supplier,
 			"doctype": "Outward Bank Payment",
 			"field_map": {
 				"supplier": "party",
-				"outstanding_amount": "amount",
```

Comment:
Why did you remove this?

Reply:
The above-mentioned statement is also applicable for this condition.